dali/base/dasds.cpp

```diff
@@ -7163,19 +7163,19 @@ void CCovenSDSManager::stop()
 
 void CCovenSDSManager::restart(IException * e)
 {
-    LOG(MCwarning, unknownJob, "-------: stopping SDS server");
+    DBGLOG("-------: stopping SDS server");
```

Comment:
Some of this should be PROGLOG so we see something when DBGLOG's are suppressed.
Make this one a PROGLOG.